.github/codecov.yml

```diff
@@ -1,11 +0,0 @@
-comment: false
```

Comment:
Can we keep this file? The GH annotations in particular are a bit annoying

Reply:
Can you share an example of the annotations you mention? 

I thought this yaml file was not being used. What components uses it?

Reply:
They look like this: https://user-images.githubusercontent.com/155044/181088096-36eb1d45-5b7d-4f4f-9721-185effbb9115.png

Reply:
Do you have one from our CI? 

How is this file being used?

Reply:
I followed BechmarkTools (https://github.com/JuliaCI/BenchmarkTools.jl/blob/master/.codecov.yml) and moved the `codecov.yml` file to the root of the project (and prepended a `.`).

Reply:
Just kidding, the [documentation](https://docs.codecov.com/docs/codecov-yaml) says that it has to be in `.git`.